README.md

```diff
@@ -160,3 +160,15 @@ This can be found in the [CHANGELOG.md](CHANGELOG.md) file.
 
 ## License
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fxmartlabs%2FBender.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2Fxmartlabs%2FBender?ref=badge_large)
+
+## Citation
+If you use this code in your research please cite us:
+
+@misc{xmartlabs-2017-bender,
+  author = {Mathias Claassen and Santiago Castro},
+  title = {Bender: Easily craft fast Neural Networks on iOS!},
```

Comment:
This it how it looks like in one of my LaTeX docs:
![image](https://user-images.githubusercontent.com/3905501/228002949-a494b3a1-7c8c-409f-901e-424a97842d67.png)

What about this change?

```suggestion
  title = {Bender: Easily craft fast Neural Networks on {iOS}!},
```